CppCoreGuidelines.md

```diff
@@ -12952,51 +12952,51 @@ Never allow an error to be reported from a destructor, a resource deallocation f
 1. `nefarious` objects are hard to use safely even as local variables:
 
 
-    void test(string& s)
-    {
-        nefarious n;          // trouble brewing
-        string copy = s;      // copy the string
-    } // destroy copy and then n
+        void test(string& s)
+        {
+            nefarious n;          // trouble brewing
+            string copy = s;      // copy the string
+        } // destroy copy and then n
 
-Here, copying `s` could throw, and if that throws and if `n`'s destructor then also throws, the program will exit via `std::terminate` because two exceptions can't be propagated simultaneously.
+    Here, copying `s` could throw, and if that throws and if `n`'s destructor then also throws, the program will exit via `std::terminate` because two exceptions can't be propagated simultaneously.
 
 2. Classes with `nefarious` members or bases are also hard to use safely, because their destructors must invoke `nefarious`' destructor, and are similarly poisoned by its poor behavior:
 
 
-    class innocent_bystander {
-        nefarious member;     // oops, poisons the enclosing class's destructor
-        // ...
-    };
+        class innocent_bystander {
+            nefarious member;     // oops, poisons the enclosing class's destructor
+            // ...
+        };
 
-    void test(string& s)
-    {
-        innocent_bystander i; // more trouble brewing
-        string copy = s;      // copy the string
-    } // destroy copy and then i
+        void test(string& s)
+        {
+            innocent_bystander i; // more trouble brewing
+            string copy2 = s;      // copy the string
```

Comment:
not sure about the renaming of `copy` to `copy2`... within this example, there is only one 'copy'.